stdlib/3.4/asyncio/tasks.pyi

```diff
@@ -24,7 +24,9 @@ def as_completed(fs: Sequence[_FutureT[_T]], *, loop: AbstractEventLoop = ...,
                  timeout: Optional[float] = ...) -> Iterator[Generator[Any, None, _T]]: ...
 def ensure_future(coro_or_future: _FutureT[_T],
                   *, loop: AbstractEventLoop = ...) -> Future[_T]: ...
-async = ensure_future
+# The following alias causes a syntax error since Python 3.7:
+# if sys.version_info < (3, 7):
+#     async = ensure_future
```

Comment:
I'm unclear on how this would help. Doesn't this still fail to parse under Python 3.7? Dead code must still be syntactically correct.

Reply:
Yes, if we remove the alias we should also remove this commented-out code.

Reply:
The comment was intended to show what ideally should be in this file (but can't). I think there should be some kind of comment to note that `async` is deliberately missing from the stub for older Python versions. But it could be as simple as:

```python
# 'async' was an alias for 'ensure_future' prior to Python 3.7.
```

(Of course if you decide not to apply this patch, we should actually add the commented out code.)

Reply:
This patch makes sense. A comment like that would be better than mysteriously commented-out code. I'd also add to the comment a mention of the fact that `async` became a keyword in 3.7.